Makefile

```diff
@@ -146,26 +146,34 @@ publish-compile: docker-build-compile
 	docker push $(COMPILE_IMAGE)
 
 docker-clean-worker:
+	rm -rf docker_build_worker.tmpdir
```

Comment:
Do we need this directory in the first place? It seems a bit atypical to me that we mount shared volume for builds.

Reply:
For posterity: there was a conversation about this in slack, and we all agreed that multi-stage builds would be better than bind mounts. @chainlink posted a solution that uses skaffold, which may be the best longer-term solution. For now, I don't think we could change this without changing our overall build process significantly.

On Slack, I believe the main reservations around bind mounts were whether they worked on docker-for-mac, and it seems that they do. Docker-for-mac seems to somehow mount host volumes into the docker-for-mac VM, and does so without changing the owner to root.

Reply:
Yup it works on mac 👍